tests/conftest.py

```diff
@@ -70,3 +73,361 @@ def _sample_all_wsis(_sample_ndpi, _sample_svs, _sample_jp2, tmpdir_factory):
         shutil.copy(_sample_jp2, dir_path.joinpath(_sample_jp2.basename))
 
     return dir_path
+
+
+@pytest.fixture(scope="session")
+def _source_image(tmpdir_factory):
+    """Sample pytest fixture for src image. Download src image image for pytest."""
```

Comment:
"""Sample pytest fixture for source image. 
Download stain normalisation source image for pytest."""

First line should be one sentence only.

Reply:
I had to break most of them into two because the flake8  tests were complaining that there were more than 88 characters in one line. If you want me, I can make them all one line.